generated/python/gapic-google-cloud-spanner-admin-database-v1/google/cloud/gapic/spanner_admin_database/v1/database_admin_client.py

```diff
@@ -471,7 +473,7 @@ def update_database_ddl(self,
             ``longrunning.Operation``: ``<database>/operations/<operation_id>``.
 
             ``operation_id`` should be unique within the database, and must be
-            a valid identifier: ``[a-zA-Z][a-zA-Z0-9_]*``. Note that
+            a valid identifier: ``[a-z][a-z0-9_]*``. Note that
```

Comment:
Did allow, or did not allow? The updated docs don't seem to permit capital letters.